extensions/OVR/OVR_multiview.txt

```diff
@@ -778,4 +777,5 @@ Revision History
                                 interop with compatibility profile's Begin/End (34). Add mention
                                 of OES_geometry_shader and OES_tessellation_shader to interop.
                                 Clarify what happens when transform feedback is paused.
+      5     07/25/18  oetuaho   Fix off-by-one issue in baseViewIndex + numViews check.
```

Comment:
Don't forget to change the revision and date at the top of the spec.

Reply:
Done.